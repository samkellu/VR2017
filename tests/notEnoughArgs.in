./binary_files/02.bin 0xAA 0x11
