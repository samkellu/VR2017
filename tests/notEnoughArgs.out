Incorrect number of command line arguments entered. Correct format: ./vr2017 <.bin file> <byte 1 of delimiter> <byte 2 of delimiter> <byte 3 of delimiter>
